pkg/csi/service/wcpguest/controller.go

```diff
@@ -311,13 +399,15 @@ func (c *controller) ControllerPublishVolume(ctx context.Context, req *csi.Contr
 	if err != nil {
 		msg := fmt.Sprintf("Validation for PublishVolume Request: %+v has failed. Error: %v", *req, err)
 		log.Error(msg)
+		volumeControlOpsHistVec.WithLabelValues("attach", "fail").Observe(time.Since(start).Seconds())
 		return nil, status.Errorf(codes.Internal, msg)
 	}
 
 	// File volumes support
 	if isFileVolumeRequest {
 		// Check the feature state for file volume support
 		if !commonco.ContainerOrchestratorUtility.IsFSSEnabled(ctx, common.FileVolume) {
+			volumeControlOpsHistVec.WithLabelValues("attach", "fail").Observe(time.Since(start).Seconds())
```

Comment:
PR release note says you are adding support for create and delete.

Reply:
This is already addressed. You are commenting on the outdated code.